post_to_mastodon.py

```
@@ -176,7 +192,38 @@ def to_unicode_bold(text: str) -> str:
+    # if we have an image path, we post the image as well
+    if args.image:
+        # first check if the image suffix is of type PNG or JPG(JPEG)
+        if not args.image.lower().endswith((".png", ".jpg", ".jpeg")):
+            logger.error(
+                "Image must be of type PNG or JPG(JPEG). " f"Received: {args.image}"
+            )
+            sys.exit(1)
+        # check whether we find an image of that name,
+        # for this we walk the current directory
+        image_path = None
+        for root, dirs, files in os.walk(os.getcwd()):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)